.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,99 @@
+name: Bug Report
+description: Report a bug encountered in ovn-kubernetes
+labels: kind/bug
+body:
+  - type: textarea
+    id: problem
+    attributes:
+      label: What happened?
+      description: |
+        Please provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner.
+        If this matter is security related, please disclose it privately via https://github.com/ovn-org/ovn-kubernetes/blob/master/SECURITY.md
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected
+    attributes:
+      label: What did you expect to happen?
+    validations:
+      required: true
+
+  - type: textarea
+    id: repro
+    attributes:
+      label: How can we reproduce it (as minimally and precisely as possible)?
+    validations:
+      required: true
+
+  - type: textarea
+    id: additional
+    attributes:
+      label: Anything else we need to know?
+
+  - type: textarea
+    id: OVN Kubernetes Version
```

Comment:
do you mean kube version? The kubectl will only give us the k8 version. We probably want k8+ovn-k version.
If we want to get the ovn-kube version, `ovnkube.sh` spits out the version at the start:
```
================== ovnkube.sh --- version: 3 ================
 ==================== command: ovnkube-controller-with-node
 =================== hostname: ovn-worker3
 =================== daemonset version 3
 =================== Image built from ovn-kubernetes ref: refs/heads/e2e-test-provider  commit: aaabb6f73c1197dc321ff7e29ffd570507401138
```

Reply:
no I meant ovnkube version; we are going to have releases moving forward. That's also why I intentionally added this field. See https://github.com/ovn-org/ovn-kubernetes/releases and https://github.com/ovn-org/ovn-kubernetes/milestones; our first one is somewhat being planned around mid May at the moment. However you do raise a good point on "how can the user know which version they are using". I might have to find a way to reflect that nicely moving forward.

Reply:
I think its good to also capture the k8 version its deployed on.

Reply:
`If we want to get the ovn-kube version, ovnkube.sh spits out the version at the start:` heh yes this!
ok let me see how this is generated and then adapt the help space to get that info.
I figured if we know the ovnkube version we can infer the ovn and k8s version on our own because we know which version will be pinned by ovnkube.

Reply:
haha see:
```
# ovnkube.sh version (update when API between daemonset and script changes - v.x.y)
ovnkube_version="3"

# The daemonset version must be compatible with this script.
# The default when OVN_DAEMONSET_VERSION is not set is version 3
ovn_daemonset_version=${OVN_DAEMONSET_VERSION:-"3"}
```
lol thanks @martinkennelly seems like when I cut the release I should do it at that mark where I do that PR, merge it then immediately release.

Reply:
ack I think then let's capture the following:
ovnkube version, kubernetes version, ovn-version and ovs version? because for all we know users could be mix and matching things here..

Reply:
@martinkennelly !!! OMG!! look what I found :)

TADA!
```
./ovnkube --version
Version: 0.3.0
Git commit: 4a8a31a13f17612fbaf3d204be1be5bc65fa44cf
Git branch: fix-templates-ovnk
Go version: go1.21.4
Build date: 2024-05-02
OS/Arch: linux amd64
```
LOOK: https://github.com/ovn-org/ovn-kubernetes/pull/329/files :) lol makes our job very easy then!
We just need good documentation TBH :)

Reply:
sweeet :)

Reply:
Thats lovely - gives us go version and os/arch.